custom_components/dreo/__init__.py

```diff
@@ -45,34 +46,51 @@ async def async_setup_entry(hass: HomeAssistant, config_entry: ConfigEntry) -> b
         return False
 
     device_dict = process_devices(manager)
+    _LOGGER.debug("Device dict is: %s", device_dict)
 
     manager.start_transport()
     
     hass.data[DOMAIN] = {}
     hass.data[DOMAIN][DREO_MANAGER] = manager
 
     fans = hass.data[DOMAIN][DREO_FANS] = []
+    heaters = hass.data[DOMAIN][DREO_HEATERS] = []
     platforms = []
 
-    _LOGGER.debug(device_dict)
+    _LOGGER.debug("Device dict: %s", device_dict)
```

Comment:
device_dict isn't modified is it?  Why log again?

Reply:
Detritus from an earlier debugging attempt. Removed.